runtime/vm/jvminit.c

```diff
@@ -2960,6 +2961,12 @@ processVMArgsFromFirstToLast(J9JavaVM * vm)
 			vm->extendedRuntimeFlags |= J9_EXTENDED_RUNTIME_POSITIVE_HASHCODE;
 		} else if (0 == strcmp(testString, VMOPT_XXDISABLEPOSITIVEHASHCODE)) {
 			vm->extendedRuntimeFlags &= ~(UDATA)J9_EXTENDED_RUNTIME_POSITIVE_HASHCODE;
+		} else if (0 == strcmp(testString, VMOPT_XXDEBUGLOCALMAPPER)) {
```

Comment:
This needs to be rebased: `testString` no longer exists.

Reply:
Agreed - it's been blocked on me finding time to write the test case